aspnetcore/security/authentication/certauth.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Configure certificate authentication in ASP.NET Core
+author: blowdart
+description: Learn how to configure certificate authentication in ASP.NET Core for IIS and HTTP.sys.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: barry.dorrans
+ms.date: 06/11/2019
+uid: security/authentication/certauth
+---
+# Overview
+
+`Microsoft.AspNetCore.Authentication.Certificate` contains an implementation similar to [Certificate Authentication](https://tools.ietf.org/html/rfc5246#section-7.4.4) for ASP.NET Core. Certificate authentication happens at the TLS level, long before it ever gets to ASP.NET Core. More accurately, this is an authentication handler that validates the certificate and then gives you an event where you can resolve that certificate to a `ClaimsPrincipal`. 
+
+[Configure your host](#configure-your-host-to-require-certificates) for certificate authentication, be it IIS, Kestrel, Azure Web Apps, or whatever else you're using.
+
+## Get started
+
+Acquire an HTTPS certificate, apply it, and [configure your host](#configure-your-host-to-require-certificates) to require certificates.
+
+In your web app, add a reference to the package. Then in the `Startup.Configure` method, call
```

Comment:
What package? We should call it out here.

Reply:
Microsoft.AspNetCore.Authentication.Certificate added